person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- **[OPTIONAL]** You need to generate the `id` and add its instance variable. You are declaring getter for id, but you did not define it anywhere. You can have add this to the initialize method

  ```
  @id = rand(1...1000)
  ```